lib/action_kit_rest/user.rb

```diff
@@ -3,5 +3,18 @@ class User < Base
     def base_path
       'user'
     end
+
```

Comment:
This is intentionally not retrieving additional phone data when listing users. It should be straightforward to retrieve the data using `ActionKitRest::Client.phone` if needed.